CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * [CHANGE] ring/client: It's now possible to set different value than `consul` as default KV store. #120
 * [CHANGE] Lifecycler: Default value of lifecycler's `final-sleep` is now `0s` (i.e. no sleep). #121
 * [CHANGE] Lifecycler: It's now possible to change default value of lifecycler's `final-sleep`. #121
+* [CHANGE] Minor cosmetic changes in ring and memberlist HTTP status templates. #149
```

Comment:
Is this really a breaking change?  Looks like the output is for humans.

Reply:
It is definitely not breaking, but it's a change. Would you prefer this to be an `[ENHANCEMENT]`?